Loaded up varef_manual_annotation_check.xml in IGV

In [8]:
!head -n 100000 /home/mcn26/varef/scripts/noon_data/1.annotate/chr22_annotated_output.csv/part-00065-9ebe92ff-c084-4000-96d6-fcf6fc2371be-c000.csv > head.csv

In [12]:
import pandas as pd

df=pd.read_csv("head.csv")

with pd.option_context('display.max_columns', None):
    display(df)

    

CHROM       POS            ID  REF ALT QUAL FILTER  \
0      chr22  10526268             .    T   C    .   PASS   
1      chr22  10565188             .    G   T    .   PASS   
2      chr22  10573104  rs1162884139    T   C    .   PASS   
3      chr22  10579899  rs1398662523    T   A    .   PASS   
4      chr22  10591003             .    A   T    .   PASS   
...      ...       ...           ...  ...  ..  ...    ...   
42107  chr22  50796441             .  GGA   G    .   PASS   
42108  chr22  50796453             .    A   T    .   PASS   
42109  chr22  50796671  rs1353688683    G   T    .   PASS   
42110  chr22  50797561             .    T   C    .   PASS   
42111  chr22  50806120             .    G   T    .   PASS   

                                                    INFO  K562__ref  \
0      K562__ref=1.0208274;HepG2__ref=1.2801021;SKNSH...   1.020827   
1      K562__ref=0.15342037;HepG2__ref=0.049771953;SK...   0.153420   
2      K562__ref=0.6979168;HepG2__ref=0.62394214;SKNS...   0.697917   
3      K562__ref=-0.0026473154;HepG2__ref=0.10845182;...  -0.002647   
4      K562__ref=0.45357764;HepG2__ref=0.4049733;SKNS...   0.453578   
...                                                  ...        ...   
42107  AN_oth=2034;AN_ami=906;AN_sas=4708;AN_fin=9788...        NaN   
42108  K562__ref=2.7041638;HepG2__ref=2.6656473;SKNSH...   2.704164   
42109  K562__ref=0.62436503;HepG2__ref=1.5405772;SKNS...   0.624365   
42110  K562__ref=0.25098827;HepG2__ref=0.20299597;SKN...   0.250988   
42111  K562__ref=0.33623913;HepG2__ref=0.35076416;SKN...   0.336239   

       HepG2__ref  SKNSH__ref  K562__alt  HepG2__alt  SKNSH__alt  K562__skew  \
0        1.280102    1.731577   1.075862    1.311914    1.777054    0.055035   
1        0.049772    0.053634   0.137038    0.071452    0.095832   -0.016382   
2        0.623942    0.698069   0.677637    0.601024    0.674310   -0.020280   
3        0.108452    0.158937  -0.004767    0.085142    0.173008   -0.002120   
4        0.404973    0.494793   0.576130    0.387677    0.463847    0.122552   
...           ...         ...        ...         ...         ...         ...   
42107         NaN         NaN        NaN         NaN         NaN         NaN   
42108    2.665647    2.664950   2.624311    2.527909    2.642365   -0.079852   
42109    1.540577    2.692317   0.652535    1.428495    2.618241    0.028170   
42110    0.202996    0.203944   0.246318    0.201056    0.198239   -0.004670   
42111    0.350764    0.692072   0.484029    0.465325    0.792194    0.147790   

       HepG2__skew  SKNSH__skew    AC      AN        AF  cadd_phred  \
0         0.031812     0.045477     1  133386  0.000007       8.438   
1         0.021681     0.042198     2   87186  0.000023       4.230   
2        -0.022918    -0.023759     3  115540  0.000026       4.726   
3        -0.023310     0.014071     3  109342  0.000027      20.500   
4        -0.017297    -0.030946     1  120962  0.000008       6.428   
...            ...          ...   ...     ...       ...         ...   
42107          NaN          NaN     1  148250  0.000007       0.676   
42108    -0.137739    -0.022585     1  150116  0.000007       1.776   
42109    -0.112082    -0.074076  1785  140284  0.012724       2.476   
42110    -0.001940    -0.005705     1  147238  0.000007       3.154   
42111     0.114561     0.100122     1  125122  0.000008       3.222   

       is_in_dELS  is_in_CA  is_in_pELS  is_in_CA-H3K4me3  is_in_CA-CTCF  \
0           False      True       False             False          False   
1           False     False       False             False          False   
2           False     False       False             False          False   
3           False     False       False             False          False   
4           False     False       False             False          False   
...           ...       ...         ...               ...            ...   
42107       False     False       False             False          False   

Let's look a a few. First: one in no annotation: chr22:10526268 T->C. All genomic loci booleans false execpt `is_in_CA`

![igv screenshot](chr22_10526268.png)

(far left, as you can see from coords in search bar).

Overlaps with an annotation...

In [14]:
!zcat /home/mcn26/varef/data/ENCODE/SCREEN_v4_cCREs_agnostic/GRCh38-cCREs.V4.bed.gz | grep EH38D6337083

chr22	10526200	10526411	EH38D6337083	EH38E4276528	CA


...And looking up the annotation, yep, it's `CA`.

Now let's look at chr22 10565188 , which is `false` in all genomic region columns. 

![another igv screenshot](chr22_10565188.png)

Again, far left. We see it's not in any annotations, as expected. 